src/_guides/libraries/objective-c-interop.md

```diff
@@ -328,6 +341,34 @@ to the correct isolate.
 For an example of this,
 see the implementation of [`package:cupertino_http`][].
 
+There are a few steps required to handle callbacks:
+
+1. Create an Objective-C function, or method, 
+that calls the method of interest. Your helper function or method 
+must include a callback that transforms the returned data 
+into Dart C Objects and sends the data, or a pointer to the object, 
+to the Dart Port. You can see an [example here](https://github.com/dart-lang/http/blob/2f8205c5254886e088aa5ebd52a2e4dc4ec18afb/pkgs/cupertino_http/src/CUPHTTPClientDelegate.m#L63).
+
+    - You should copy the Dart API classes from the [`dart-sdk` directory](https://github.com/dart-lang/http/tree/master/pkgs/cupertino_http/src/dart-sdk).
```

Comment:
The idea is that this whole section is just temporary but can't say for sure how long. If we break it out into its own page then I think it makes sense to go into more details on why its necessary and what the classes do. 

As for licensing requirements, I am not really sure. Ideally there's a better way to do this (like importing a library or something), but since this is meant to be a temporary solution I don't think the team is looking to do much more work. @liamappelbe any thoughts?

Reply:
Google has a policy of putting code that's under a different license in a third_party folder.

https://opensource.google/documentation/reference/thirdparty/non-google3

Yes, I'd also prefer a separate page.